### PR TITLE
Include motivation and responsiveness formula for resource-constrained devices

### PR DESCRIPTION
**Pull request by bjj-te**
Changes as distributed on the IETF IPPM mailing list and discussed in the RPM community meeting on January 10, 2024.

Happy to drop the local development tweaks commit before merging, but am keeping it there so I can build locally if any changes are requested.

### CODE REVIEW

**Comment by hawkinsw** [comment]
Thank you @bjj-te -- we will review ASAP!

Will

**Comment by cpaasch** [comment]
Looks good to me! Thanks @bjj-te ! Giving this a few more days for others to chime in and then I will merge.

**Review by cpaasch — APPROVED** [review]

File: draft-ietf-ippm-responsiveness.md [hunk 1]
```diff
@@ -512,6 +528,20 @@ Responsiveness = 60000 /
 
 This responsiveness value presents round-trips per minute (RPM).
 
+#### For the TCP-Only Case
+
+If there are no TLS connections being used, then the notion of a normalised round-trip time for the TLS handshake does not apply.
+Zeroes cannot be substituted for `tls_f`, as that will result in an artificially low responsiveness value.
+Instead, the same principle of weighing each contribution to the foreign RTT equally (and then weighting the foreign and self RTTs
```
Comment (bjj-te) [inline draft-ietf-ippm-responsiveness.md:535]: Just spotted this typo - will push a new commit to fix...

**Comment by bjj-te** [comment]
I've just force-pushed a very small wording change to avoid the fact that I am not sure whether "weighting" is a verb or a noun:

```diff
-Instead, the same principle of weighing each contribution to the foreign RTT equally (and then weighting the foreign and self RTTs
-equally) is applied.
+Instead, the same principle of giving each contribution to the foreign RTT equal weight, and then giving the foreign and self RTTs
+equal weights is applied.

**Comment by cpaasch** [comment]
Thanks Ben! I'm merging your PR.

**Pull request merged**